pages/account/manage/event/[[...data]].js

```diff
@@ -340,10 +336,11 @@ export default function ManageEvent({ BASE_URL, event }) {
                       onTagAdd={handleTagAdd}
                       onTagRemove={handleTagRemove}
                       tags={tags}
-                      inputRef={tagInputRef}
+                      showNotification = {showNotification}
+                        setShowNotification = {setShowNotification}
```

Comment:
I think this indentation is not correct?

Reply:
should i fix this and update the branch ?

Reply:
yes please 👍

Reply:
Prettier plugin in VScode will fix this automatically

Reply:
done